HOMEWORKS/Chapter_05/Saeid/assets/js/script.js

```diff
@@ -0,0 +1,234 @@
+
+  let cardFive = document.createElement("div"); // card 05
+  cardFive.className = "card"; // Create attribute class
+  let photoFive = document.createElement("img"); // Create image
+  photoFive.src = "/HOMEWORKS/Chapter_05/assets/img/img_05.jpg"; // Image url
+  cardFive.appendChild(photoFive);
+
+  let cardSix = document.createElement("div"); // card 06
+  cardSix.className = "card"; // Create attribute class
+  let photoSix = document.createElement("img"); // Create image
+  photoSix.src = "/HOMEWORKS/Chapter_05/assets/img/img_06.jpg"; // Image url
+  cardSix.appendChild(photoSix);
+
+  photo.appendChild(cardOne); // card 01
+  photo.appendChild(cardTwo); // card 02
+  photo.appendChild(cardThree); // card 03
+  photo.appendChild(cardFour); // card 04
+  photo.appendChild(cardFive); // card 05
+  photo.appendChild(cardSix); // card 06
+  divOfPageWrapper.appendChild(photo); // Image section
+};
+photoSection();
+// End
+
+//Start download section
+const downloadSection = () => {
+  let download = document.createElement("section"); // create download section
+  download.id = "download_section"; // Create attribute id
+
+  let downTitle = document.createElement("p"); // create download title
+  TitleOfDown = document.createTextNode("Free Download at Start bootstrap!"); //title text
+  downTitle.appendChild(TitleOfDown);
+
+  let downButton = document.createElement("button"); // create header button
+  buttonOfDown = document.createTextNode("DOWNLOAD NOW!"); // button text
+  downButton.appendChild(buttonOfDown);
+
+  download.appendChild(downTitle); // title
+  download.appendChild(downButton); // button
+  divOfPageWrapper.appendChild(download); // download section
+};
+downloadSection();
+// End
+
+//Start footer
+const footerSection = () => {
+  let footer = document.createElement("footer"); // create footer section
+  footer.id = "footer_section"; // Create attribute id
+
+  let containerOfFooter = document.createElement("div"); // create container for content
+  containerOfFooter.className = "container"; // Create attribute class
+
+  let footerTitle = document.createElement("h1"); // create title
+  textOfFooterTitle = document.createTextNode("Let's Get In Touch!"); // title text
+  footerTitle.appendChild(textOfFooterTitle);
+
+  let footerLine = document.createElement("hr"); // create line under title
+
+  let footerContent = document.createElement("p"); // create content
+  footerContent.innerHTML =
+    "<p>Ready to start your next project with us? That's great! Give us a call or send us an email and we <br> will get back to you as soon as possible!</p>";
+
+  let cardOfTell = document.createElement("div"); // card of phone number
+  cardOfTell.className = "card fas fa-phone";
+  cardOfTell.innerHTML = "<p>123-456-6789</p>";
+
+  let cardOfMail = document.createElement("div"); // card of email
+  cardOfMail.className = "card far fa-envelope";
+  cardOfMail.innerHTML = "<p>feedback@startbootstrap.com</p>";
```

Comment:
Also here i want you to make a function that validate email address.
validateEmail('feedback@startbootstrap.c') return "invalid@email.wrong"
If valid then it will return the same email. You have to check about  the formula like this:
STRING@STRING.com
So you should check for @ and .com that are exists in the email. otherwise return "invalid@email.wrong"